search-engines.lisp

```diff
@@ -1074,6 +1074,130 @@ REQUEST-ARGS is a list of args to pass to request function."
      :completion-function (make-pkgs-completion)
      :documentation "UNIX/GNU/Linux package search."))
 
+(define-search-engine peertube
```

Comment:
Out of interest, is Peertube somehow related to YouTube? If their queries are similar, we can hit two birds with one stone there :)

Reply:
No, PeerTube is a completely different service altogether and they don't share any functionality. It's a decentralized video hosting platform where one can host their videos in their own instance. Also, see note in my comment below.